packages/liquidator/test/Liquidator.js

```diff
@@ -143,7 +143,7 @@ contract("Liquidator.js", function(accounts) {
         await collateralToken.approve(emp.address, convert("100000000"), { from: liquidatorBot });
         await collateralToken.approve(emp.address, convert("100000000"), { from: liquidityProvider });
 
-        syntheticToken = await Token.at(await emp.tokenCurrency());
+        syntheticToken = (await Token.at(await emp.tokenCurrency())).contract;
```

Comment:
I never ran into this bug is this neccessary?

Reply:
I'm not sure if the truffle instance has a `methods` object. However, even if it does, passing only the web3 contract means that the tests won't allow us to make the same mistake in the future (using truffle calls rather than web3 ones). Thoughts?

Reply:
ok cool, I didn't realize that the truffle contrcat has a web3 contract under `contract`. Should we make this change in other parts of the bot tests?

Reply:
Potentially. Hopefully, this will plug the hole for now, though.